packages/newspringchurchapp/src/prayer/UserPrayerList/index.js

```diff
@@ -48,77 +48,80 @@ class UserPrayerList extends React.Component {
         <FlexedSafeAreaView forceInset={{ top: 'always' }}>
           <ScrollView>
             <Query query={GET_USER_PRAYERS} fetchPolicy="cache-and-network">
-              {({ data: { userPrayers = [] } = {} }) => (
-                <>
-                  <PaddedView>
-                    <H6>Viewing</H6>
-                    <GreenH4>My Prayers</GreenH4>
-                  </PaddedView>
-                  <Mutation
-                    mutation={DELETE_PRAYER}
-                    update={(cache, { data: { deletePrayer } }) => {
-                      const data = cache.readQuery({
-                        query: GET_USER_PRAYERS,
-                      });
-                      const { id } = deletePrayer;
-                      const updatedPrayers = data.userPrayers.filter(
-                        (prayer) => prayer.id !== id
-                      );
-                      cache.writeQuery({
-                        query: GET_USER_PRAYERS,
-                        data: { userPrayers: updatedPrayers },
-                      });
-                    }}
-                  >
-                    {(deletePrayer) => (
-                      <StyledView>
-                        {userPrayers && userPrayers.length > 0 ? (
-                          userPrayers.map((prayer) => (
-                            <Card key={prayer.id}>
-                              <CardContent>
-                                <PrayerSingle
-                                  prayer={prayer}
-                                  showDate
-                                  action={
-                                    <ActionComponent
-                                      component={<DeleteIcon />}
-                                      options={[
-                                        {
-                                          title: 'Delete Prayer',
-                                          method: async () => {
-                                            await deletePrayer({
-                                              variables: {
-                                                parsedId: prayer.id,
-                                              },
-                                            });
+              {({ loading, data: { userPrayers = [] } = {} }) => {
+                if (loading) return <ActivityIndicator />;
```

Comment:
just preference really but this one could probably use a `PaddedView`, it's very high on the screen

Reply:
I agree with this. It is pretty high up on the screen. Moving it down a bit will put it in the users line of sight which makes it look more intentional than accidental.

Reply:
We could just center it, if that's the desired look.

Reply:
![Screen Shot 2019-07-30 at 12 14 53 PM](https://user-images.githubusercontent.com/8878532/62146591-b36ad280-b2c3-11e9-9f02-fb0c539c6b6e.png)

This is where it's currently positioned.

Reply:
love it

Reply:
I'm not seeing this as centered. It's still way high in the top for me. Was there styling that just didn't get pushed or something?